CHANGELOG.rst

```diff
@@ -13,6 +13,12 @@ Change Log
 Unreleased
 ~~~~~~~~~~
 
+[4.0.0] - 2021-01-28
+~~~~~~~~~~~~~~~~~~~~
+
+* Add CodeAnnotationChecker to run generic checks on annotations
+* BREAKING CHANGE: modify the numerical ID of annotation checks
```

Comment:
Should the new checks also be mentioned under the breaking change?  I wonder if any new linting checks should be considered a breaking change, because if someone has a threshold of zero and new violations without amnesty, it will break their build. Food for thought.

Reply:
I _think_ that new checks should not be considered as breaking changes... But I'm not sure.

Reply:
Technically, I agree with you. From the perspective of someone pulling in an upgrade and wondering whether it might break their build, I think it would be nice to warn them. Given that this already is a major version bump, they will likely read the change log, and they may just concentrate on the `BREAKING CHANGE` section.

Here is a proposed update:
```
* BREAKING CHANGES:
  
  * modify the numerical ID of annotation checks
  * though technically not a breaking change, the new annotation checks may break your build if there are pre-existing violations.
```